components/eam/bld/build-namelist

```diff
@@ -817,11 +822,14 @@ add_default($nl,'use_rad_dt_cosz');
 #if 0., no rayleigh friction is applied
 add_default($nl, 'raytau0');
 
-# these options werre added for global RCE and they only work with RRTMGP
-my $rad_pkg = $cfg->get('rad');
-if ($rad_pkg eq 'rrtmgp') {
-  add_default($nl,'do_aerosol_rad');
-  add_default($nl,'fixed_total_solar_irradiance');
+# these options were added for global RCE and they only work with RRTMGP
+my $rad_pkg = 'none';
+if (!$ideal_or_adia_or_aqua) {
```

Comment:
shouldn't we allow RRTMGP settings for aqua planet?

Reply:
yea, this logic seems wrong. It seems better to just have a block to disable rad for ideal physics

Reply:
Good catch, this was a mistake, will fix.